lib/pbench/test/unit/agent/test_tool_group.py

```diff
@@ -0,0 +1,147 @@
+import os
+import pytest
+
+from pathlib import Path
+
+from pbench.agent.tool_group import BadToolGroup, ToolGroup
+
+# from pbench.common.exceptions import BadConfig
+
+
+class TestToolGroup:
+
+    pbench_run = "/mock/pbench_run"
+    group = "tool-group"
+    mock_tg_dir_name = Path(pbench_run, f"tools-v1-{group}")
```

Comment:
The `tools-v1` prefix should come from the `ToolGroup.TOOL_GROUP_PREFIX` for most use cases, and then verify that the old `tools-` prefix is not considered.

And probably add a case for a blatantly "bad" prefix.